components/proxy/plugins/corsorigin/go.mod

```diff
@@ -3,6 +3,97 @@ module github.com/gitpod-io/gitpod/proxy/plugins/corsorigin
 go 1.17
 
 require (
-	github.com/caddyserver/caddy/v2 v2.4.3
+	github.com/caddyserver/caddy/v2 v2.4.5
 	github.com/rs/cors v1.8.0
 )
+
+require (
```

Comment:
Why does materializing all references instead of just `require github.com/caddyserver/caddy/v2 v2.4.3` work now? Because we're not no longer on a fork? Just trying to understand. :-)

Reply:
This is a change in [Go 1.17](https://golang.org/doc/go1.17#go-command) :(